keycloak-user-api/src/main/java/io/enmasse/user/keycloak/KeycloakUserApi.java

```diff
@@ -586,7 +586,7 @@ static User buildUser(UserRepresentation userRep, List<GroupRepresentation> grou
             } else if (groupRep.getName().contains("_")) {
                 String[] parts = groupRep.getName().split("_");
```

Comment:
I think it would be simpler to use as limit of 2 here:

~~~java
String[] parts = groupRep.getName().split("_", 2);
~~~

Reply:
Fixed